src/integrationTest/java/com/mongodb/hibernate/query/AbstractQueryIntegrationTests.java

```diff
@@ -187,6 +187,23 @@ protected void assertActualCommand(BsonDocument expectedCommand) {
                 .containsAllEntriesOf(expectedCommand);
     }
 
+    protected void assertMutationQuery(
+            String hql,
+            int expectedMutationCount,
+            String expectedMql,
+            MongoCollection<BsonDocument> collection,
+            Iterable<? extends BsonDocument> expectedDocuments,
+            Set<String> expectedAffectedCollections) {
+        assertMutationQuery(
+                hql,
+                mutationQuery -> {},
```

Comment:
`assertSelectionQuery` and `assertSelectQueryFailure` pass `null` as a no-op `queryPostProcessor`. And `assertMutationQuery` is already written to handle `null` values for this parameter. Let's pass `null` here similarly to other places.

Alternatively we can refactor all this code to avoid handling and passing `null`s, but I don't think it is worth the change.